marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/entitySearch/getMinAndMaxPropertyValues.api

```diff
@@ -0,0 +1,21 @@
+{
+  "functionName": "getNumericFacetsRange",
```

Comment:
For simplicity and consistency, the functionName should match the name of the endpoint file.